src/graph/utils/ocl_usm_utils.hpp

```diff
@@ -30,6 +30,9 @@ namespace ocl {
 void *malloc_shared(
         cl_device_id dev, cl_context ctx, size_t size, size_t alignment = 0);
 
+void *malloc_device(
+        cl_device_id dev, cl_context ctx, size_t size, size_t alignment = 0);
```

Comment:
Where's this function used? The below benchdnn usage is for SYCL runtime, but this defined function is for OCL runtime?

Reply:
Thanks for catching it. Updated the commit.